examples/directml/llm/llm.py

```diff
@@ -39,16 +39,24 @@ def set_config_parameters(repo_id: str, num_layers: Optional[int]):
         config.state_dict = pipeline.model.state_dict()
 
     config.hidden_size = llm_model.config.hidden_size
-    config.intermediate_size = llm_model.config.intermediate_size
     config.num_heads = llm_model.config.num_attention_heads
-    config.num_key_value_heads = llm_model.config.num_key_value_heads
     config.num_layers = num_layers or llm_model.config.num_hidden_layers
     config.vocab_size = llm_model.config.vocab_size
     config.model_type = main_model.config.model_type
     config.apply_residual_connection_post_layernorm = getattr(
         llm_model.config, "apply_residual_connection_post_layernorm", True
     )
 
+    if repo_id == "tiiuae/falcon-7b-instruct":
+        config.intermediate_size = llm_model.config.hidden_size * 4
+    else:
+        config.intermediate_size = llm_model.config.intermediate_size     
```

Comment:
I think it would be good to make it less specific and more robust by using similar logic to `hasattr` below. If `config.intermediate_size` is missing, we can assume that the intermediate size is going to be `llm_model.config.hidden_size * 4`. It will likely change in the future but for the models that we currently support it should work (and hopefully will work in a similar way for models that we onboard in the future).